### DISCUSSION TITLE
Arrows on edges, how do we add them and style them?

### DISCUSSION THREAD

**Discussion by joshuacalloway** [open-source-labs/Svelvet#460] [Q&A]
I don't see a way to add arrows to edges.  Was this removed?
I want to be able to create two nodes and visualize a 

    [ parent ] ----> [ child ] 

relationship

**Comment by bgins** [comment]
I was trying to find out what happened as well. It seems like history where arrows existed must have been rewritten. The first commit available in the repo is https://github.com/open-source-labs/Svelvet/commit/207aec8d9f63e5fa52da69246982c0c81055f33d from March of this year, but this library has been around for over a year.

The API doesn't appear to support arrows on edges. I started experimenting a bit with a custom edge with a `marker-end`, but so far this does not render an arrow:

```svelte
<script>
  import { Edge } from "svelvet";

  const edgeColor = "#777777";
</script>

<defs>
  <marker
    id="arrow"
    markerWidth="9"
    markerHeight="9"
    refX="6.5"
    refY="3.5"
    orient="auto"
  >
    <polygon points="0 0, 7 3.5, 0 7" fill={edgeColor} />
  </marker>
</defs>

<Edge let:path>
  <path d={path} />
</Edge>

<style>
  path {
    stroke: #777777;
    stroke-width: 2px;
    marker-end: "url(#arrow)";
  }
</style>
```

I would also love to be able to use arrows. Will continue experimenting and report back if I get anything working. Open to suggestions here of course as well. 🙂
